storage/tokudb/ha_tokudb_alter_56.cc

```diff
@@ -248,6 +248,40 @@ static bool only_flags(ulong bits, ulong mask) {
     return (bits & mask) != 0 && (bits & ~mask) == 0;
 }
 
+// Table create options that should be ignored by TokuDB
+// There are totally 25 create options defined by mysql server(see handler.h),
```

Comment:
Space after "server", and I think "are totally 25" -> "are 25 ... in total" but George should know better